### ISSUE TITLE
Problems saving UTF-8 characters

### ISSUE THREAD

**Issue by i3visio** [chfw/pyexcel#12]
We are trying to make this code work using special characters:
```
from pyexcel_ods import save_data
data = {}
data.update({"Sheet 1": [[1, 2, 3], [4, 5, 6]]})
data.update({"Sheet 2": [[u"row 1", u"Héllô!", u"HolÁ!"]]})
save_data("your_file.ods", data)
```
Unfortunately, this throws the following exception:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/pyexcel_ods/__init__.py", line 296, in save_data
    write_data(afile, data, file_type=file_type, **keywords)
  File "/usr/local/lib/python2.7/dist-packages/pyexcel_io/__init__.py", line 256, in store_data
    writer.write(data)
  File "/usr/local/lib/python2.7/dist-packages/pyexcel_io/base.py", line 258, in write
    sheet.write_array(sheet_dicts[name])
  File "/usr/local/lib/python2.7/dist-packages/pyexcel_io/base.py", line 222, in write_array
    self.write_row(r)
  File "/usr/local/lib/python2.7/dist-packages/pyexcel_ods/__init__.py", line 257, in write_row
    self.write_cell(tr, x)
  File "/usr/local/lib/python2.7/dist-packages/pyexcel_ods/__init__.py", line 246, in write_cell
    tc.setAttrNS(OFFICENS, x_odf_value_token, x)
  File "/usr/local/lib/python2.7/dist-packages/odf/element.py", line 440, in setAttrNS
    self.attributes[(namespace, localpart)] = c.convert((namespace, localpart), value, self)
  File "/usr/local/lib/python2.7/dist-packages/odf/attrconverters.py", line 1488, in convert
    return conversion(attribute, value, element)
  File "/usr/local/lib/python2.7/dist-packages/odf/attrconverters.py", line 77, in cnv_double
    return str(arg)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 1: ordinal not in range(128)
```
Is there something we are missing when writing down Unicode characters? This is important in lots of languages which have characters that go beyond basic ASCII.

**Comment by chfw** [comment]
This is a known defect, please refer to https://github.com/chfw/pyexcel-ods/issues/2. I am finding time to resolve it.

**Comment by chfw** [comment]
Alternatively, you could use pyexcel-ods3:    

    # -*- coding: utf-8 -*-
    from pyexcel_ods3 import save_data
    data = {}
    data.update({"Sheet 1": [[1, 2, 3], [4, 5, 6]]})
    data.update({"Sheet 2": [[u"row 1", u"Héllô!", u"HolÁ!"]]})
    save_data("your_file.ods", data)

**Comment by chfw** [comment]
Marking this duplicate and please following pyexcel-ods issues for an update

**Comment by chfw** [comment]
Please try it out by:

    pip install git+https://github.com/chfw/pyexcel-ods.git

**Comment by i3visio** [comment]
Oh! I have just tested it and it works! That's a great job man!

**Comment by i3visio** [comment]
Just a question to release it in production: will be upgraded this improvement to pip?

**Comment by chfw** [comment]
released to pypi as 0.0.9. pip shall be able to get it on fresh installation. for existing one, use the following commands:


     pip install pyexcel-ods==0.0.9

or

    pip install --upgrade pyexcel-ods
